resources/lang/en/api.php

```diff
@@ -33,5 +33,9 @@
         'friends' => [
             'read' => 'See who you are following.',
         ],
+
+        'users' => [
+            'read' => 'Read the public profile of any user.',
```

Comment:
...on your behalf.

Can also use some rearrangements as it may sound a bit weird.